git-branchless/src/opts.rs

```diff
@@ -420,6 +418,11 @@ pub enum Command {
         /// that; otherwise, the editor starts empty.
         #[clap(action, short = 'd', long = "discard", conflicts_with("messages"))]
         discard: bool,
+
+        /// A commit to "fix up". The reworded commits will become `fixup!` commits (suitable for
+        /// use with `git rebase --autosquash`) targetting the supplied commit.
```

Comment:
```suggestion
        /// A commit to "fix up". The reworded commits will become `fixup!` commits (suitable for
        /// use with `git rebase --autosquash`) targeting the supplied commit.
```

I think this message is pretty much fine as-is; either you're familiar with `--autosquash` and you know what this option does, or you'll need to read more about it anyways.